helpers/setup-sa.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
```

Comment:
What's the purpose of this file? Looks like it's copied from project-factory and isn't relevant here.